canonical-web-and-design/maas-ui
# Replace the run script with the dotrun snap

## Done
* Remove the `./run` script
* Document how to run the project via the dotrun snap
* Add `start` NPM script to support the snap 

## QA
* Run `./run` and see it fails
* Follow the README and install the snap
* Join the VPN
* Ensure the config is pointing at karura
* Run `dotrun` from the root of the project
* Check it installs correctly
*  Go to http://localhost:8400/MAAS/r
* Login with the karura login details
* See that its working from within the docker container
* Swich to a local maas and check that works too
